src/core/p5.Element.js

```diff
@@ -1001,26 +1001,22 @@ p5.Element.prototype.drop = function(callback, fxn) {
 
   // Is the file stuff supported?
   if (window.File && window.FileReader && window.FileList && window.Blob) {
-    // If you want to be able to drop you've got to turn off
-    // a lot of default behavior
-    attachListener(
-      'dragover',
-      function(evt) {
-        evt.stopPropagation();
-        evt.preventDefault();
-      },
-      this
-    );
+    if (!this._dragDisabled) {
+      this._dragDisabled = true;
```

Comment:
can you clarify where this._dragDisabled is used?

Reply:
that's all there is. it's just to ensure that the `preventDefault` stuff is only done once per element.

Reply:
ah got it, thanks